deployments/nimbus-k8tls/templates/daemonset.yaml

```diff
@@ -21,4 +21,11 @@ spec:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+          - name: K8TLS_NAMESPACE
+            value: {{ include "nimbus-k8tls.fullname" . }}-env
+          {{- if .Values.output.elasticsearch.enabled }}
+          - name: OUTPUT
+            value: "ELASTICSEARCH"
```

Comment:
If the config map is loaded into the job, what is the need for environment variables ?